director_test.go

```diff
@@ -188,7 +188,7 @@ func TestQueryDirector(t *testing.T) {
 	expectedLocation := "http://redirect.com"
 	handler := func(w http.ResponseWriter, r *http.Request) {
 		w.Header().Set("Location", expectedLocation)
-		w.WriteHeader(http.StatusFound)
+		w.WriteHeader(http.StatusTemporaryRedirect)
```

Comment:
I'm trying to understand why this is changed from StatusFound to StatusTemporaryRedirect? This is probably my lack of http understanding, but I'd appreciate a clarification.

Reply:
The director has always returned a `307: Temporary Redirect`. The reason I had to change this was that I noticed the old code never actually checked the status code returned by the director. Now the client's director-handling code returns an error if the status code is not the expected 307. Since this test was mocking the director's response, it needed to be updated with the correct status code.